src/libraries/System.Private.CoreLib/src/System/Globalization/GregorianCalendarHelper.cs

```diff
@@ -35,6 +35,7 @@ internal EraInfo(int era, int startYear, int startMonth, int startDay, int yearO
             this.yearOffset = yearOffset;
             this.minEraYear = minEraYear;
             this.maxEraYear = maxEraYear;
+            // codeql[cs/leap-year/unsafe-date-construction-from-two-elements] - DateTime is constructed using the user specifed values, not a combination of different sources.
```

Comment:
The data is read from the registry on Windows and from ICU on non-Windows. I think we need to update the description.

Reply:
```suggestion
            // codeql[cs/leap-year/unsafe-date-construction-from-two-elements] - DateTime is constructed using consistent data read from the registry, not a combination of different sources.
```